app/frontend/src/locationAutocomplete.js

```diff
@@ -4,13 +4,11 @@ import { locations } from './search/data/locations';
 $(document).ready(function () {
```

Comment:
Do we not want to move away from a jQuery dependency?

Reply:
yes this was a bit lazy, im going to leave it for the moment however as this is a part of modernizing the rest if the app JS